### PR TITLE
feat(console): sort connectors

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

List connectors in preset orders.

The unknown connector will be placed in the last.

## Testing

截屏2022-12-05 下午4 27 49

### CODE REVIEW

File: packages/console/src/pages/Connectors/components/CreateForm/index.tsx [hunk 1]
```diff
@@ -56,7 +57,14 @@ const CreateForm = ({ onClose, isOpen: isFormOpen, type }: Props) => {
...
         })),
       }))
-      .filter(({ connectors }) => !connectors.every(({ added }) => added));
+      .filter(({ connectors }) => !connectors.every(({ added }) => added))
+      .slice()
+      .sort((connectorA, connectorB) => {
+        const orderA = getConnectorOrder(connectorA.target, connectorA.isStandard);
+        const orderB = getConnectorOrder(connectorB.target, connectorB.isStandard);
+
+        return orderA - orderB;
```
Comment (darcyYe) [inline packages/console/src/pages/Connectors/components/CreateForm/index.tsx:66]: nit: the result of `Number.POSITIVE_INFINITY - Number.POSITIVE_INFINITY` is `NaN`, will that affect the order result?

File: packages/console/src/pages/Connectors/components/CreateForm/constants.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  'apple',
+  'facebook',
+  'github',
+  'discord',
+  'wechat',
+  'alipay',
+  'kakao',
+  'naver',
+  'azuread',
+];
```
Comment (simeng-li) [inline packages/console/src/pages/Connectors/components/CreateForm/constants.ts:12]: should it be placed somewhere common? Or console is fine?

Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/CreateForm/constants.ts:12]: LOG-4865 the "list" should be placed somewhere too

File: packages/console/src/pages/Connectors/components/CreateForm/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/CreateForm/index.tsx:66]: updated to a safer value.

**Review by darcyYe — APPROVED** [review]

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**